doc/sign_requests.md

```diff
@@ -0,0 +1,142 @@
+Sign requests
+==================
+
+Sign requests are used to prepare document to signing and send it to signers.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Create Sign Request](#create-sign-request)
+- [Get all Sign Requests](#get-all-sign-requests)
+- [Get Sign request by ID](#get-sign-request-by-id)
+- [Cancel Sign Request](#cancel-sign-request)
+- [Resend Sign Request](#resend-sign-request)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+Create Sign Request
+------------------------
+
+The [`createSignRequest(BoxAPIConnection api, List<BoxSignRequestSigner> signers, List<BoxSignRequestFile> files, String parentFolderId)`][create-sign-request]
+method will create sign request.
+
+You can provide signers that need to sign the request and files from which sign document will be created. You can use [`BoxSignRequestSigner`][box-sign-request-signer] 
+and [`BoxSignRequestFile`][box-sign-request-file] classes to fill the necessary data.
+
+<!-- sample create_sign_request -->
+```java
+List<BoxSignRequestSigner> signers = new ArrayList<BoxSignRequestSigner>();
+BoxSignRequestSigner newSigner = new BoxSignRequestSigner("signer@mail.com");
+signers.add(newSigner);
+
+List<BoxSignRequestFile> files = new ArrayList<BoxSignRequestFile>();
+BoxSignRequestFile file = new BoxSignRequestFile("12345");
+files.add(file);
+// you can also use specific version of the file
+BoxSignRequestFile file = new BoxSignRequestFile("12345", "");
+
+String parentFolderId = "55555";
+
+BoxSignRequest.Info signRequestInfo = BoxSignRequest.createSignRequest(api, signers,
+        files, parentFolderId);
+```
+
+Create sign request allow you to specify optional parameters using the [`BoxSignRequestCreateParams`][sign-request-create-params]
+object.
+
+```java
+List<BoxSignRequestSigner> signers = new ArrayList<BoxSignRequestSigner>();
+BoxSignRequestSigner newSigner = new BoxSignRequestSigner("signer@mail.com");
+signers.add(newSigner);
+
+List<BoxSignRequestFile> files = new ArrayList<BoxSignRequestFile>();
+BoxSignRequestFile file = new BoxSignRequestFile("12345");
+files.add(file);
+
+String parentFolderId = "55555";
+
+BoxSignRequestCreateParams createParams = new BoxSignRequestCreateParams()
+        .setIsDocumentPreparationNeeded(true);
+
+BoxSignRequest.Info signRequestInfo = BoxSignRequest.createSignRequest(api, signers,
+        files, parentFolderId, createParams);
+```
+
+If you set ```isDocumentPreparationNeeded``` flag to true, you need to visit ```prepareUrl``` before the sign request will be sent.
```

Comment:
Can we elaborate on what `prepareUrl` is and what role it plays in the signing process?